react/react.py

```diff
@@ -96,7 +98,7 @@ def to_json(data):
         :return: Received data in json format
         :rtype: str
         """
-        if isinstance(data, str):
+        if isinstance(data, six.string_types):
```

Comment:
> Checking for six.string_types instead of six.text_type will break the current behaviour for anyone passing bytes (in py3) to that function.

Nevermind, `six.string_types` is `str` in py3. Don't change it.